### ISSUE TITLE
GIF not working

### ISSUE THREAD

**Issue by LukeDuckwalker** [TwidereProject/Twidere-Android#1233]
## Expected behavior
GIF images shown in the TimeLine and reproduced if clicked.

## Actual behavior
No media shown, tweets with GIFs attached shown as only text.

## Steps to reproduce
See any tweet with GIFs attached

## Extra info

Android version: 9

Network type:

- [x] Cellular
- [x] Wi-Fi

App version: 4.0.3

Build variant:

- [x] Google
- [ ] F-Droid

Micro-blogging service:

- [x] Twitter
- [ ] Fanfou
- [ ] GNU Social / StatusNet
- [ ] Mastodon

App settings:

- [ ] Custom API
- [ ] Custom consumer key/secrets
- [ ] [Official keys](https://gist.github.com/mariotaku/5465786)
- [ ] Using [Thumbor](https://github.com/thumbor/thumbor)
- [ ] Connected through proxy
- [ ] Media preview enabled

**Comment by Tedko** [comment]
similar to #1070

**Comment by Tlaster** [comment]
It happens when using the official key, will looking into it since the official key is now back online

**Comment by LukeDuckwalker** [comment]
Thanks for the info! Hope it can be solved 👍 :-)

**Comment by foreheadlarge** [comment]
It happens when using the official keys of iPhone and iPad.

**Comment by Tedko** [comment]
Seems like the IAP functionalities of send twitter w Giphy is not working as well. --

**Comment by Tedko** [comment]
> Seems like the IAP functionalities of send twitter w Giphy is not working as well. --

Update - no problem at all

**Comment by Tedko** [comment]
just try tested on twidere seems gif is working, try click on Paper Clip logo when compose tweet and `Add GIF`. If you encounter any bug also try use the official key (logoff and login with `+` - `Twitter` -> `Twitter` ) @LukeDuckwalker @foreheadlarge 

Have a try with official key & take a look of this tweet? https://twitter.com/suji_yan/status/1255355070390112256  of twitter.com/suji_yan

**Comment by foreheadlarge** [comment]
Twidere v4.1.1/Twitter for iPhone

**Comment by foreheadlarge** [comment]
![Screenshot_20200429-223403_1](https://user-images.githubusercontent.com/63905609/80604590-dd2c7200-8a6c-11ea-900e-51a8271b2fd0.png)
9

**Comment by Tedko** [comment]
umm, wondering are you using custom API/keys or official keys?

**Comment by foreheadlarge** [comment]
Yes, it's the official key of the iPhone.  This problem also occurs in v3.7.8.

On the other hand, the official keys for GIFs to display correctly are Android and Mac, etc.

**Comment by Tedko** [comment]
That’s really strange. Any idea of why? 🤔

**Comment by foreheadlarge** [comment]
Sorry, I have no idea.
